### ISSUE TITLE
[Writing] Case Study: HPE

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#3623]
Contributors and community members are encouraged to participate in Layer5's [Writing Program](https://layer5.io/community/handbook/writing-program)). A new written piece fitting the following description is requested to be written by a community member:

**Desired Title:** HPE uses Meshery to integrate SPIRE into Istio
**Topic:** 
**Core points to convey in the post:** 
1. Engage with Maximiliano
 Churchi  and Yogi Porla for quotes and materials. 
1. Use the materials from the [DockerCon 2022 talk and blog post](https://layer5.io/blog/docker/extending-docker-with-meshery-spire-and-istio).

See https://www.hashicorp.com/case-studies as examples. Don't work about the actual PDF creation - just about the content.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="16px" align="left" /><h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

**Comment by deek121477** [comment]
Can I work on this issue?

**Comment by deep-poharkar** [comment]
can you assign me this?

**Comment by Shivam-AfA** [comment]
We'll let @deek121477 go first.

**Comment by deek121477** [comment]
> We'll let @deek121477 go first.

Thank you..

**Comment by withgaurav** [comment]
I have written a blog on this topic. @leecalcote

**Comment by leecalcote** [comment]
@withgaurav thanks, but no thanks. We’re looking for a case study. 

@deek121477 hows it coming along? Not hearing from you and not seeing a draft PR in a couple of days, let’s remove your assignment, so that others can make progress here.

**Comment by deek121477** [comment]
> @withgaurav thanks, but no thanks. We’re looking for a case study.
> 
> @deek121477 hows it coming along? Not hearing from you and not seeing a draft PR in a couple of days, let’s remove your assignment, so that others can make progress here.

Yes sir. I have written the blog, but not drafted a PR yet. I will do it by today.

**Comment by leecalcote** [comment]
Draft completed - https://docs.google.com/document/d/1QdSxNaJBgiB_xG3KIvTqfoUCEXp8B8F8yoAvY0AOoaM/edit
